File: server/oracli_api/models/mentee.py
from oracli_api.models import user

import uuid

import oracli_api


class Mentee(user.User):
    def __init__(self, name, age, gender, email=None, password=None):
        super().__init__(name, age, gender, email, password)
        self.is_mentor = False
        self.mentor = None
        self.why_mentor = ""

    def serialize(self):
        return {
            "name": self.name,
            "age": self.age,
            "is_mentor": self.is_mentor,
            "gender": self.gender,
            "email": self.email,
            "password": self.password,
            "about": self.about_you,
            "interests": self.areas_of_interest,
            "mentor": self.mentor,
            "why_mentor": self.why_mentor
        }

    def save_new(self):
        oracli_api.mentees.insert_one(self.serialize())

File: mobile/oracli/Models/Mentee.swift
//
//  Mentee.swift
//  oracli
//
//  Created by Anika Morris on 2/7/20.
//  Copyright © 2020 Anika Morris. All rights reserved.
//

import Foundation

struct Mentee:Codable {
    
    let name: String
    let age: Int
    let gender: String
    let email: String
    
    let token: String?
    let interests: String?
    let whyMentor: String?
    let about: String?
    let mentor: Mentor?
}
